site/content/blogs/release-v116.ja.md

```diff
@@ -0,0 +1,184 @@
+# AWS Copilot v1.16: マルチパイプライン、 SNS サブスクリプションフィルターなど
+
+The AWS Copilot コアチームは Copilot v1.16 リリースを発表できることを嬉しく思います。
+このリリースに貢献した 7 人の新しいコミュニティ開発者を歓迎します。[@codekitchen](https://github.com/codekitchen),
+[@shingos](https://github.com/shingos) 、 [@csantos](https://github.com/csantos) 、 [@rfma23](https://github.com/rfma23)、
+[@g-grass](https://github.com/g-grass) 、 [@isleys](https://github.com/isleys) 、
+[@kangere](https://github.com/kangere) 。 [パブリックコミュニティチャット](https://gitter.im/aws/copilot-cli) は成長していて、270 人以上の方がオンラインで、日々助け合ってます。また、 最近 [GitHub](http://github.com/aws/copilot-cli/) で 2.1 K を超える Star を獲得しました。
+AWS Copilot に関心とサポートを示してくださった皆様に感謝しています。
+
+
+Copilot v1.16 にはいくつかの新機能と改善が加えられました:
+
+* **マルチパイプライン:** `copilot pipeline init` は、リポジトリ内の別々のブランチをつ席する複数の AWS CodePipeline を作成します。詳細は、
+[こちら](./#create-multiple-pipelines-per-branch)を確認してください。
+* **SNS サブスクリプションフィルターポリシー:** `filter_policy` フィールドを利用して、 Worker Service は、サブスクライブしたトピックに対する SNS メッセージをフィルターできます。
+詳細は、[こちら](./#define-messages-filter-policy-in-publishsubscribe-service-architecture)を確認してください。
+* **その他 たくさんの改善点:**
+    * `deploy` コマンドに `--no-rollback` フラグが追加されました。デプロイが失敗した場合にスタックの自動ロールバックを無効化します。([#3341](https://github.com/aws/copilot-cli/pull/3341)). 新しいフラグは、インフラストラクチャの更新失敗時にトラブルシューティングをする際に役立ちます。例えば、デプロイが失敗した場合、CloudFormation はスタックをロールバックするため、失敗時に発生したログを削除します。このフラグは、問題のトラブルシューティングの為にログを保存するのに役立ちます。保存後にロールバックし、マニフェストを再度更新できます。
+    * `package` コマンドに `--upload-assets` フラグが追加されました。CloudFormation テンプレートを生成する前に、ECR または S3 にアセットをプッシュします ([#3268](https://github.com/aws/copilot-cli/pull/3268))。 このフラグによりパイプラインの buildspec を大幅に簡略化ができます。 buildspec を再作成する場合は、ファイルを削除し、`copilot pipeline init` を再度実行します。その際に、プロンプトでは、既存のパイプライン名を指定します。
+    * Environment で `task run` 実行中にセキュリティグループを追加できるようになりました ([#3365](https://github.com/aws/copilot-cli/pull/3365))。
+    * Copilot が CI 環境で実行されている場合(環境変数 `CI` を `true` を設定している場合)に、Docker 進行状況の更新ノイズが少なくなりました ([#3345](https://github.com/aws/copilot-cli/pull/3345))。
+    * App Runner がサポートされていないリージョンに対して、App Runner Service をデプロイするときに警告ログを出力するようになりました([#3326](https://github.com/aws/copilot-cli/pull/3326))。
+    * `app show` および `env show` コマンドは、テーブル形式で Service や Job がデプロイされた Environment を出力するようになりました。([#3379](https://github.com/aws/copilot-cli/pull/3379)、[#3316](https://github.com/aws/copilot-cli/pull/3316))。
+    * パイプラインマニフェストの `build.buildspec` を使って buildspec パスをカスタマイズできるようになりました。([#3403](https://github.com/aws/copilot-cli/pull/3403)).
```

Comment:
I think the period of the sentence "(https://github.com/aws/copilot-cli/pull/3403)." should be Japanese one ("。").

Reply:
Yes, I fixed it.